File: cpp/0026_RemoveDuplicatesFromSortedArray.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int res=0;
        int label=-101;
       
        for(int i=0;i<nums.size();i++)
        {
            while(i<nums.size()&&nums[i]==label)
            {
                 i++;
            }
            if(i>=nums.size())
                break;
             nums[res]=nums[i];
             label=nums[i];
             res++;
            
            
        }
        return res;
        
    }
};

File: python/0026_RemoveDuplicatesFromSortedArray.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        res=1
        for i in range(1,len(nums)):
            if(nums[i]==nums[i-1]):
                i+=1
            else:
                nums[res]=nums[i]
                res+=1

        return res

